src/Microsoft.HttpRepl.Tests/HttpStateTests.cs

```diff
@@ -49,6 +53,146 @@ public void GetPathString_MultiplePathSections_CorrectString()
             Assert.Equal(expected, result);
         }
 
+        [Fact]
+        public void GetApplicableContentTypes_NoBaseAddress_ReturnsNull()
+        {
+            HttpState httpState = SetupHttpState();
+
```

Comment:
nit: don't need the empty line here.  Usually an empty line implies a pivot in focus (i.e. I did one thing now I'm doing another), or in tests it's the bridge from A-A-A.  It actually is a little distracting reading this, as my immediate sensation is that the Act portion of this test is to set httpState.BaseAddress to null - I have to read the rest of the test to understand that this is still part of the Arrangement.

Reply:
Fixed in a973059.